electron-app/src/main.ts

```diff
@@ -185,24 +177,25 @@ function createWindow() {
   }
 
   mainWindow.webContents.on('new-window', event => event.preventDefault());
-  mainWindow.on('closed', () => {
-    mainWindow = null;
-    if (global.tray && util.isWindows()) {
-      clearTimeout(backgroundAlertTimeout);
-      if (!hasNotifiedAboutBackground) {
-        backgroundAlertTimeout = setTimeout(() => {
-          hasNotifiedAboutBackground = true;
-          const notification = new Notification({
-            title: 'PostyBirb',
-            icon,
-            body: 'PostyBirb will continue in the background.',
-            silent: true,
-          });
-          notification.show();
-        }, 750);
+  if (!global.settingsDB.getState().quitOnClose)
```

Comment:
I feel like the 'closed' event listener's instantiation shouldn't be affected by the setting. It would be better to just stick the 'if' inside and do an app quit when true. This would support better behavior imo and would allow the app to instantly take any updates to the setting into account when the user then closes the window.

Reply:
Thats right! I will update that